.gitignore

```diff
@@ -13,3 +13,6 @@ astroid.egg-info/
 .cache/
 .eggs/
 .pytest_cache/
+.python-version
```

Comment:
@hippo91 These should go into your global gitignore, as they are specific to tools you use on your dev environment rather than specific to tools used by the project itself.

Reply:
Yes ! Off-course.